app/lib/frontend/static_files.dart

```diff
@@ -124,46 +127,29 @@ class StaticFile {
   );
 }
 
-/// The static assets that get a ?hash=<hash> in their URL and with that, longer
-/// cached timeouts.
-final hashedFiles = const <String>[
-  'css/github-markdown.css',
-  'css/style.css',
-  'highlight/github.css',
-  'highlight/highlight.pack.js',
-  'highlight/init.js',
-  'img/atom-feed-icon-32x32.png',
-  'img/dart-logo-400x400.png',
-  'img/dart-logo.svg',
-  'img/dart-packages.png',
-  'img/dart-packages-white.png',
-  'img/flutter-packages.png',
-  'img/flutter-packages-white.png',
-  'img/pub-dev-logo-2x.png',
-  'js/gtag.js',
-  'js/script.dart.js',
-];
-
 class StaticUrls {
```

Comment:
Is this simply exposing the static urls being hosted by a `StaticFileCache`? If so, why doesn't the constructor take a specific `StaticFileCache` instance? I'm guessing their are all singletons... Maybe we could hide the constructors? At mark them visibleForTestsOnly...